sr_robot_launch/scripts/sr_ur_arm_unlock

```diff
@@ -0,0 +1,166 @@
+        rospy.sleep(5)
+        for arm in arms:
+            self.call_arm_service(arm, "brake_release", Trigger)
+        rospy.sleep(5)
+
+    def check_arms_e_stops(self, arms):
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.ROBOT_EMERGENCY_STOP:
+                while safety_mode.safety_mode.mode == SafetyMode.ROBOT_EMERGENCY_STOP:
+                    rospy.logwarn("Emergency stop button is still pressed for arm %s, please release", arm)
+                    safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+                    rospy.sleep(2)
+
+    def check_fix_arms_protective_stop(self, arms):
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.PROTECTIVE_STOP:
+                rospy.loginfo("Protective stop detected on: %s", arm)
+                self.call_arm_service(arm, "unlock_protective_stop", Trigger)
+
+    def check_fix_arms_fault(self, arms):
+        fault = False
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.FAULT:
+                fault = True
+                rospy.loginfo("Fault detected on: %s", arm)
+                rospy.loginfo("This can be caused by using the external e-stop. Have you released it?")
+                self.call_arm_service(arm, "restart_safety", Trigger)
+        return fault
+
+    def check_arms_robot_mode(self, arms):
+        arm_needs_starting = False
+        for arm in arms:
+            robot_mode = self.call_arm_service(arm, "get_robot_mode", GetRobotMode)
+            if robot_mode.robot_mode.mode == RobotMode.IDLE or robot_mode.robot_mode.mode == RobotMode.POWER_OFF:
+                rospy.loginfo("Starting arm: %s", arm)
+                arm_needs_starting = True
+        return arm_needs_starting
+
+    def clear_arms_popups(self, arms):
+        for arm in arms:
+            self.call_arm_service(arm, "close_safety_popup", Trigger)
+        rospy.sleep(2)
+        for arm in arms:
+            self.call_arm_service(arm, "close_popup", Trigger)
+        rospy.sleep(2)
+
+    def check_fix_program_loaded_arms(self, arms):
+        sleep_time = False
+        for arm in arms:
+            if not rospy.get_param("/" + arm + "_sr_ur_robot_hw/headless_mode"):
+                play_msg = self.call_arm_service(arm, "program_state", GetProgramState)
+                if play_msg.program_name == "null":
+                    rospy.loginfo("Not in headless mode. Loading program: %s for arm: %s", self.external_control_program_name, arm)
+                    resp = self.call_arm_service(arm, "load_program", Load)
+                    rospy.loginfo("%s", resp)
```

Comment:
remove debug code

Reply:
It is pretty verbose, but I think it's useful to see the arms response to a load_program request. Do you think I should remove it?

Reply:
ok, lets keep it then